wafer/sponsors/templates/wafer.sponsors/sponsors.html

```diff
@@ -6,16 +6,21 @@
 <div class="wafer list">
     {% for sponsor in sponsor_list %}
     <section class="wafer wafer-sponsor">
-    <div>
-        <h2>
-           {% if sponsor.url %}
-           <a href="{{ sponsor.url }}">{{ sponsor.name }}</a>
-           {% else %}
-           {{ sponsor.name }}
-           {% endif %}
-        </h2>
-        {{ sponsor.description.rendered|safe }}
+    <div class="well">
+      <h2 class="bs-callout bs-callout-default bs-callout-{{ sponsor.packages.first.name.lower }}">
+         {% if sponsor.packages.first.name.lower == 'platinum' %}
```

Comment:
Yeah, I agree. 

I mentioned it in [my commit message](https://github.com/CTPUG/wafer/pull/277/commits/811426f2f1b1048c782c249a5895e9eb0f94d7ca)